include/cling/Interpreter/InvocationOptions.h

```diff
@@ -36,6 +36,11 @@ namespace cling {
     void Parse(int argc, const char* const argv[],
                std::vector<std::string>* Inputs = nullptr);
 
+    ///\brief Whether the options given will not override anything for the
```

Comment:
The options given where? Who will override? What's "for the Interpreter's default language options"? What does "false" mean given "whether... will not"? I.e. could you expand yet simplify this doc a bit? :-)